src/main/java/jp/ac/keio/bio/fun/xitosbml/TrialRunner.java

```diff
@@ -0,0 +1,22 @@
+package jp.ac.keio.bio.fun.xitosbml;
+
+public class TrialRunner {
+
+	public static void run(String path1, String path2) {
+		// imager.show();
+		new CuiMainImgSpatial().runCui(path1, path2);
+	}
+
+	public static void main(String[] args) {
+		String path1 = "C:\\Users\\Subroto\\Desktop\\Inference(300x100)\\inf_18.tif";
+		String path2 = "C:\\Users\\Subroto\\Desktop\\Inference(300x100)\\inf_18.xml";
+		// "C:\\Users\\Subroto\\Desktop\\Inference(300x100)\\gt_5_yo.xml";
+		/*
+		 * if (path.contains(".")) path2 = path.substring(0, path.indexOf('.')); // new
+		 * ij.ImageJ(); ImagePlus imager = new ImagePlus(path); // img.show(); String
+		 * title = imager.getTitle(); System.out.println(title);
+		 * System.out.println(path2);
+		 */
+		run(path1, path2);
+	}
```

Comment:
Why don't we just implement this as the main method in `CuiMainImageSpatial`?

Reply:
I don't think this will work as the main method because essentially, we are not taking input from the command line in this case. Though if you find this more appropriate, we can make this the main method.

Reply:
I don't think I fully understand. Wouldn't it be doing essentially the same thing except in a different class?

Reply:
Are you asking about CuiRun.java and TrialRunner.java? I used TrialRunner.java for only figuring out how to use the file path to eventually get an SBML document instead of using the dialog box to get an image. Currently, TrialRunner.java has no use, it is purely ornamental, and independent of other classes since we use CuiRun.java as the main method.
As far as CuiRun.java and TrialRunner.java are concerned, there is no difference except for the fact the file path is hardcoded here. So, there is no requirement to take any input from the command line. As long as we do not manually change the path in the program itself, it will operate on the same image time and again. 
Regarding implementing the main method, is it possible to implement CuiRun.java as the main method in CuiMainImageSpatial?
I separated everything because I wanted to keep the structure somewhat similar to the actual XitoSBML code. I can make changes as you would wish for.

Reply:
> Currently, TrialRunner.java has no use, it is purely ornamental, and independent of other classes since we use CuiRun.java as the main method.

So is TrialRunner.java for testing only? If so do we need to add this to the repo?

> I separated everything because I wanted to keep the structure somewhat similar to the actual XitoSBML code. I can make changes as you would wish for.

As for `CuiMainImageSpatial` and `CuiMainSpatial `, you don't need to completely keep the structure and should make the architecture much simpler. XitoSBML has different options besides creating a model from images so it has an abstract class and all that. But for the CUI version,  we don't need the abstraction.

For `CuiRun` and `CuiMainImageSpatial`, we don't need to implement the main method in `CuiMainImageSpatial`. It was my misunderstanding.  Keep the main method in CuiRun.

Reply:
Removed TrialRunner.java in the new branch xitosbml_cui. 
Added new issue https://github.com/Medha-B/XitoSBML/issues/8 for CuiMainImageSpatial and CuiMainSpatial.